xbmc/GUIInfoManager.cpp

```diff
@@ -3944,6 +3944,7 @@ const infomap listitem_labels[]= {{ "thumb",            LISTITEM_THUMB },
                                   { "hastimerschedule", LISTITEM_HASTIMERSCHEDULE },
                                   { "hasrecording",     LISTITEM_HASRECORDING },
                                   { "isrecording",      LISTITEM_ISRECORDING },
+                                  { "hasarchive",       LISTITEM_HASARCHIVE },
```

Comment:
Please add documentation for the new listitem above the `listitem_labels` table.

https://github.com/xbmc/xbmc/pull/14736/files#diff-127e5f2786eb19ac8e0e2f04b3bf8306R3797

Reply:
Done

Reply:
Thanks.